hbgd_data_store_server/studies/management/commands/load_idx.py

```diff
@@ -61,7 +61,7 @@ def get_domain_variable(row, domain, variable_cache=None):
     code_idx = DOMAIN_CODE_FORMAT.format(domain=domain.code)
     cat_idx = DOMAIN_CAT_FORMAT.format(domain=domain.code)
 
-    code = row[code_idx]
+    code = row.get(code_idx, None)
```

Comment:
Shouldn't the `row` always have the code? If the row is missing the code then it might be better to let this throw an exception. I believe we encountered this situation recently with some incorrect IDX data that user was trying to load.

Reply:
I think you're right. Reverted.